### PR TITLE
Beta and maskable icons

### PR DESCRIPTION
**Pull request by alsturm**

### CODE REVIEW

File: src/assets/favicon/manifest.json [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
+			"purpose": "any"
 		},
 		{
-			"src": "icon_192x192.png",
-			"sizes": "192x192",
-			"type": "image/png"
+			"src": "icon_512x512_maskable.png",
+			"sizes": "512x512",
+			"type": "image/png",
+			"purpose": "maskable"
```
Comment (taulinger) [inline src/assets/favicon/manifest.json:13]: Hmm, the 192px image should be still referenced here

File: test/e2e/favicon.spec.js [hunk 2]
```diff
@@ -8,22 +8,26 @@ test.describe('favicons', () => {
...
+					purpose: 'any'
 				},
 				{
-					src: 'icon_192x192.png',
-					sizes: '192x192',
-					type: 'image/png'
+					src: 'icon_512x512_maskable.png',
+					sizes: '512x512',
+					type: 'image/png',
+					purpose: 'maskable'
```
Comment (taulinger) [inline test/e2e/favicon.spec.js:30]: Let's restore the test for the 192px image and additionally add the test for the maskable 512px image

**Review by taulinger — CHANGES_REQUESTED** [review]

File: test/e2e/favicon.spec.js (hunk 2 above)
Comment (alsturm) [reply test/e2e/favicon.spec.js]: done

**Review by taulinger — APPROVED** [review]
THX

**Pull request merged**